core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroup.java

```diff
@@ -65,7 +65,6 @@
 public final class HealthCheckedEndpointGroup extends DynamicEndpointGroup {
 
     static final Backoff DEFAULT_HEALTH_CHECK_RETRY_BACKOFF = Backoff.fixed(3000).withJitter(0.2);
-    static final HealthCheckStrategy DEFAULT_HEALTH_CHECK_STRATEGY = new AllHealthCheckStrategy();
```

Comment:
It was really critical bug.  `DEFAULT_HEALTH_CHECK_STRATEGY` was shared with all of HealthCheckEndpointGroup event it's stateful.

Reply:
:scream_cat: